src/core/server/models/tenant/helpers.ts

```diff
@@ -98,3 +98,10 @@ export function supportsMediaType(
       return !!tenant.media?.giphy.enabled && !!tenant.media.giphy.key;
   }
 }
+
+export function isAMPEnabled(tenant: Pick<Tenant, "featureFlags" | "amp">) {
+  if (tenant.amp !== undefined) {
+    return !!tenant.amp;
+  }
```

Comment:
Maybe this reads a bit cleaner?

```suggestion
  if (typeof tenant.amp === "boolean") {
    return tenant.amp;
  }
```